arctic/store/_version_store_utils.py

```diff
@@ -76,6 +79,87 @@ def _define_compat_pickle_load():
     return functools.partial(pickle_compat.load, compat=True)
 
 
+def analyze_symbol(l, sym, from_ver, to_ver, do_reads=False):
```

Comment:
can we add a docstring of some sort here to describe the purpose and functionality of this function?

Reply:
sure, docstrings added for all new util methods